agent_api/src/test/java/collectors/WebRequestCollectorTest.java

```diff
@@ -135,7 +135,7 @@ void testReport_ipBlockedUsingLists() {
         WebRequestCollector.Res response = WebRequestCollector.report(contextObject);
 
         assertNotNull(response);
-        assertEquals("Your IP address is not allowed. (Your IP: 192.168.1.1)", response.msg());
+        assertEquals("Your IP address is blocked. Reason: geoip restrictions (Your IP: 192.168.1.1)", response.msg());
```

Comment:
in other agents we use `due to`, which reads a bit more like a sentence. Not saying it should be the same, just for your information!

Reply:
haha yeah I know, just think this is easier for future, like "not in allowlists", instead of due to not being in allowlists for example.